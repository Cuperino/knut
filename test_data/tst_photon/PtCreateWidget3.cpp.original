void foo()
{
    PtWidget_t *containerOS_widgetMpeg;
    nargs = 0;
    PtSetArg(&args[nargs++], Pt_ARG_POS, &pos, 0);
    PtSetArg(&args[nargs++], Pt_ARG_DIM, &taille, 0);
    containerOS_widgetMpeg = PtCreateWidget(PtOSContainer, ABW_HCaScript, nargs, args);
}

void bar()
{
    PtWidget_t bouton[5];
    for (int i = 0; i < 5; ++i) {
        nargs = 0;
        PtSetArg( &args[nargs++], Pt_ARG_POS, &position, 0);
        PtSetArg( &args[nargs++], Pt_ARG_DIM, &taille, 0);
        PtSetArg( &args[nargs++], Pt_ARG_MARGIN_HEIGHT, 0, 0);
        PtSetArg( &args[nargs++], Pt_ARG_MARGIN_WIDTH, 0, 0);
        PtSetArg( &args[nargs++], Pt_ARG_BEVEL_WIDTH, 0, 0);
        bouton[i] = PtCreateWidget( BoutonDS, ABW_HCaScript, nargs, args);
    }
}
