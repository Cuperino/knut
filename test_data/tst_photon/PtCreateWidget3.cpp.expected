void foo()
{
    QFrame *containerOS_widgetMpeg = new QFrame(ABW_HCaScript);
    nargs = 0;
    containerOS_widgetMpeg->move(pos);
    containerOS_widgetMpeg->resize(&taille);
}

void bar()
{
    Photon::Widget bouton[5];
    for (int i = 0; i < 5; ++i) {
        bouton[i] = new KDAB_FIX_UNKNOWN_CLASS_BoutonDS(ABW_HCaScript);
        nargs = 0;
        bouton[i]->move(position);
        bouton[i]->resize(&taille);
        bouton[i]->setMarginHeight(0);
        bouton[i]->setMarginWidth(0);
        bouton[i]->setBevelWidth(0);
    }
}
